src/utils/validators.tsx

```diff
@@ -104,6 +104,8 @@ const badStatusHandler = (familyCode: HTTPFamilyResponseStatusCode) =>
           utils.url.redirectWithOutcome(
             actualCode === 401
               ? OUTCOME_ROUTE.AUTH_ERROR
+              : actualCode === 422
+              ? OUTCOME_ROUTE.CONFLICT
```

Comment:
Could you rewrite this condition avoiding a nested ternary please? A status to outcome mapping would be awesome